Lengnick-Style Model/lengnick_baseline.nlogo

```diff
@@ -86,7 +86,7 @@ to set-constants
   set ξ 0.01  ; A firm will switch firms based on price if the new firm is at least this fraction cheaper
   set month-length 21
   set FIRM-STRUCTURE table:get (table:from-json-file "Firm-structure.json") "Firms"
-  set n-firms table:get (table:from-json-file "Firm-structure.json") "Total firms"
+  set n-firms 30     ;PREVIOUS IMPLEMENTATION: table:get (table:from-json-file "Firm-structure.json") "Total firms" (I don't understand why this is a table to begin with)
```

Comment:
This is a table so that we can change how many firms of each type, and how many types of firm just by changing the input json file.

Reply:
Just changed this back. Sorry for the confusion, I wrote that when I first started running the model and figuring out the JSON input, and then forgot to change it back.